File: Dart_Language/14_Inheritance/Program3.java
class Parent {

		static void printData(){
			System.out.println("Java");
		}
}
class Child extends Parent {
		static void printData(){
			System.out.println("Flutter");
		}
}

class Client{
	public static void main(String[] args)	{
		Child obj = new Child();
		obj.printData();
		Child.printData();

	}
}

File: Dart_Language/15_Polymorphism/Program3.dart
class Parent {
	void career(){
		print("Engg.");
	}
	void marry(){
		print("Deepika Padukone");
	}
}
class Child extends Parent {
	void marry(){
		print("Disha");
	}
}

void main(){
	Child obj = new Child();
	obj.career();
	obj.marry();
}
